File: src/main/java/com/dingdo/RobotApplication.java
package com.dingdo;

import org.mybatis.spring.annotation.MapperScan;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

/**
 * 一些声明信息
 *
 * @author slamacraft
 * @date: 2020/8/10 15:12
 * @since JDK 1.8
 */
@MapperScan(basePackages = {"com.dingdo.**.mapper"})
@SpringBootApplication
public class RobotApplication {

    public static void main(String[] args) {
        SpringApplication springApplication = new SpringApplication(RobotApplication.class);
        springApplication.run(args);
    }
}

File: src/main/java/com/dingdo/RobotApplication.kt
package com.dingdo

import org.mybatis.spring.annotation.MapperScan
import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.EnableConfigurationProperties

@MapperScan(basePackages = ["com.dingdo.**.mapper"])
@SpringBootApplication
@EnableConfigurationProperties
open class RobotApplication {
}

fun main(args: Array<String>) {
    val springApplication = SpringApplication(RobotApplication::class.java)
    springApplication.run(*args)
}
